CONTRIBUTING.rst

```diff
@@ -27,6 +27,8 @@ branch.
 
 Ensure that the tests are all passing, and that the coverage remains at 100%.
 
+Ensure that the code the quality checks by running `nox --session lint`.
```

Comment:
```suggestion
Ensure that the code the quality checks by running ``nox --session lint``.
```

The double backticks are needed in rst to render this as inline literal.

Reply:
Working at Read the Docs, one would guess I should know better 😅 fixing